qemu/tests/cfg/live_backup.cfg

```diff
@@ -38,3 +38,12 @@
                     granularity = 32768
                 - 131072:
                     granularity = 131072
+        - data_plane:
+            only Host_RHEL.m7
```

Comment:
@xiywang Windows is supported, here 'Host_RHEL.m7' means host env, only support rhel7 host.